s3/replication/replicator/src/s3replicator/replicator_routes.py

```diff
@@ -35,12 +36,21 @@
 @routes.get('/jobs')  # noqa: E302
 async def list_jobs(request):
     """List all in-progress jobs."""
+    url = request.path_qs
+    url_ob = urlparse(url)
+    query = parse_qs(url_ob.query, keep_blank_values=True)
+
     _logger.debug('API: GET /jobs')
     jobs = request.app['all_jobs']
+    completed_jobs = request.app['completed_jobs']
 
-    _logger.debug('Number of jobs in-progress {}'.format(jobs.count()))
-    # _logger.debug('List of jobs in-progress {}'.format(Jobs.dumps(jobs)))
-    return web.json_response(jobs, dumps=Jobs.dumps, status=200)
+    if 'all' in query:
```

Comment:
when we want count for specific type, we can have following query params
- `?count`
Returns count of all jobs.

- `?count&completed` 
Returns completed count

- `?count&inprogress` 
Returns inprogress count

When `count` query param is not present, it will be treated as job listing API.